app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,24 @@
+class PostsController < ApplicationController
+  def show
+    @post = Post.find(params[:id])
+  end
+
+  def new
+  end
+
+  def create
+    @post = Post.new(post_params)
+ 
+    if @post.save
+      redirect_to @post
+        flash[:notice] = 'Your post was succefully created'
+    else
+      render 'new'
```

Comment:
I would probably display some error here as well. You are validating the presence of title and text.

Reply:
We now added an error and tested sad path